libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -1,3 +1,5 @@
+// +build unit integration
```

Comment:
@urso My idea was testing.go per package. I assume the outcome would be the same as with `helper_test.go`.